Components/Ratings/rating.html

```diff
@@ -0,0 +1,172 @@
+              style="
+                background-color: #fafafa;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+
+          <div class="page-nav">
+            <a
+              class="prev-page"
+              href="/Components/TextUtilities/text-utilities.html"
+              ><span class="material-icons"> keyboard_arrow_left </span>Text
+              Utilities</a
+            >
+            <a class="next-page" href="/Components/Lists/lists.html"
+              >list-item-inline<span class="material-icons">
+                keyboard_arrow_right
+              </span></a
```

Comment:
Could running beautify or prettier on VS code on this make it more aesthetic here? (just nitpicking)
Onkar Deshpande